synchronization/src/androidTestCyface/java/de/cyface/synchronization/SyncAdapterTest.java

```diff
@@ -108,7 +123,10 @@ public void testOnPerformSync() throws NoSuchMeasurementException, CursorIsNullE
             client = contentResolver.acquireContentProviderClient(getGeoLocationsUri(AUTHORITY));
             final SyncResult result = new SyncResult();
             Validate.notNull(client);
-            syncAdapter.onPerformSync(account, new Bundle(), AUTHORITY, client, result);
+
+            final Bundle testBundle = new Bundle();
+            testBundle.putString(MOCKED_IS_PERIODIC_SYNC_DISABLED_FALSE, "");
```

Comment:
see onPerformSync in SyncAdapter. Seems like a more or less clean mock injection as I cannot change the onPerformSync interface.

Reply:
Yeah. The usual Android problem. I know. Everything involving the Android core framework is not very testable.